.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!--
+Thank you for your pull request. Please provide a description above and review
+the requirements below.
+
+Contributors guide: https://github.com/probot/probot.github.io/blob/master/CONTRIBUTING.md
+-->
+
+##### Checklist
+<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->
+
+- [ ] The file you added does need to go in the /_apps/ folder before this can be merged.
```

Comment:
> The file you added does need to go in the /_apps/ folder before this can be merged.

Can this maybe be:

> If you're adding a listing for your app, be sure your app is in the `/_apps/` folder before opening this Pull Request